src/data/faq_de.json

```diff
@@ -54,7 +54,8 @@
                         "textblock": [
                             "Mit Version 2.2 bietet die Corona-Warn-App eine Funktion zur technischen Problem-Analyse. Fehlerberichte soll den Entwicklern helfen, schneller eine Ursachen-Analyse zu ermöglichen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei der technischen Hotline (national: 0800 7540001 bzw. international: +49 30 498 75401), um zu prüfen, ob es ggf. schon eine Lösung dafür gibt. Ein Fehlerberichte wird nur benötigt, wenn das CWA-Team danach fragt.",
                             "<ol><li>In einem solchen Fall öffnen Sie bitte ein Issue auf GitHub und beschreiben Sie das fehlerhafte Verhalten.</li><li>Starten Sie die Aufzeichnung in der CWA (---LINK oder Screenshot hier: Wie starte ich die Aufzeichnung?---)<br>- Versuchen Sie, das fehlerhafte Verhalten zu reproduzieren bzw. die Fehlermeldung zu erzeugen.<br>- Ist das gelungen, senden Sie den Fehlerbericht an den CWA-Server. Die Daten werden dabei anonymisiert und verschlüsselt. (Bei eigenem Interesse können Sie den Bericht auch lokal auf Ihrem Smartphone abspeichern.)<br>- Als Rückmeldung erhalten Sie eine eindeutige Kennung, die Sie auch in der Liste aller IDs finden <br>- Stoppen Sie die Fehleranalyse.</li><li>Aktualisieren Sie das GitHub-Issue mit Ihrer eindeutigen Kennung vom Server, damit der aufgezeichnete Bericht Ihrem Issue zugeordnet werden kann.</li></ol>",

```

Comment:
This is still a placeholder, which should be removed or filled with content:

"---LINK oder Screenshot hier: Wie starte ich die Aufzeichnung?---"